packages/js/src/core/oidc-config.test.ts

```diff
@@ -0,0 +1,26 @@
+import { fetchOidcConfig } from './oidc-config';
+
+describe('fetchOidcConfig', () => {
+  test('should return OidcConfigResponse with camel case keys', async () => {
+    const requester = jest.fn().mockResolvedValue({
+      authorization_endpoint: 'foo',
+      token_endpoint: 'foo',
+      userinfo_endpoint: 'foo',
+      end_session_endpoint: 'foo',
+      revocation_endpoint: 'foo',
+      jwks_uri: 'foo',
+      issuer: 'foo',
+    });
+    await expect(
+      fetchOidcConfig('https://example.com/oidc/.well-known/openid-configuration', requester)
+    ).resolves.toEqual({
+      authorizationEndpoint: 'foo',
+      tokenEndpoint: 'foo',
+      userinfoEndpoint: 'foo',
+      endSessionEndpoint: 'foo',
+      revocationEndpoint: 'foo',
+      jwksUri: 'foo',
+      issuer: 'foo',
+    });
```

Comment:
How about defining a mocked snake case key object, and use camelcase-keys to convert it to camel case key object? So that we can avoid creating these two objects twice.
```typescript
const  mockedOidcResponse = {
  authorization_endpoint: 'foo',
  token_endpoint: 'foo',
  userinfo_endpoint: 'foo',
  end_session_endpoint: 'foo',
  revocation_endpoint: 'foo',
  jwks_uri: 'foo',
  issuer: 'foo',
};

const requester = jest.fn().mockResolvedValue(mockedOidcResponse);

await expect(
  fetchOidcConfig('https://example.com/oidc/.well-known/openid-configuration', requester)
).resolves.toEqual(camelcaseKeys(mockedOidcResponse));
```

Reply:
If I use `camelcaseKeys` here too, both implementation and unit tests of `fetchOidcConfig` do the same thing (run same code).

And then we cannot reach the goal of testing the implementaion.